### ISSUE TITLE
Postgres Adapter

### ISSUE THREAD

**Issue by ekryski** [feathersjs/feathers#152]

**Comment by dbkaplun** [comment]
+1

**Comment by daffl** [comment]
Instead of making a database specific adapter we decided to make adapters per ORM. Here are the ones that support PostgreSQL:

- https://github.com/feathersjs/feathers-knex
- https://github.com/feathersjs/feathers-sequelize
- https://github.com/feathersjs/feathers-waterline (still in the works)

**Comment by ekryski** [comment]
I would consider this guy closed now.

**Comment by DISKONEKTeD** [comment]
Excuse the ignorance, but does this mean something like node-postgres is not compatible with feathersjs? I been looking into frameworks to help build an API quickly (instead of just going the vanilla express route).  Today I decided to delve into the basics guide for feathersjs and I did notice the support for ORMs and whatnot (personally never used an ORM and I don't plan to, due to reading up on their limitations today). But for those who want to use actual SQL queries, is that possible still?
